packages/cxl-ui/src/components/cxl-playbook-accordion.js

```diff
@@ -0,0 +1,156 @@
+import { customElement } from 'lit-element';
+import '@conversionxl/cxl-lumo-styles';
+import '@vaadin/vaadin-accordion';
+import '@vaadin/vaadin-checkbox';
+import { registerStyles } from '@vaadin/vaadin-themable-mixin/register-styles.js';
+import { registerGlobalStyles } from '@conversionxl/cxl-lumo-styles/src/utils';
+import cxlPlaybookAccordionGlobalStyles from '../styles/global/cxl-playbook-accordion-css.js';
+import { CXLVaadinAccordion } from './cxl-vaadin-accordion';
+import vaadinAccordionPanelStyles from '../styles/cxl-playbook-accordion/vaadin-accordion-panel-css.js';
+import vaadinCheckboxStyles from '../styles/cxl-playbook-accordion/vaadin-checkbox-css.js';
+
+/**
+ * Allows opening multiple panels simultaneously.
+ * Saves panel state, restores on page load.
+ */
+@customElement('cxl-playbook-accordion')
+export class CXLPlaybookAccordion extends CXLVaadinAccordion {
+  /**
+   *
+   * @return {NodeListOf<Element>}
+   */
+  get accordionPanels() {
+    return this.querySelectorAll('vaadin-accordion-panel');
+  }
```

Comment:
But we already have `items` property from upstream. What is this getter extra for?

https://github.com/vaadin/vaadin-accordion/blob/v2.0.0-alpha5/src/vaadin-accordion.js#L99-L109

Reply:
@lkraav checked that, but it doesn't work as expected: https://www.loom.com/share/85b525bf111e4b1face584e34fd53c9e Probably a lifecycle issue, but I'm not an expert on `Polymer Components`.

Reply:
Fixed, with some thought help from https://polymer.slack.com/archives/C6ULJ2F7S/p1614342362000900

You should join this Slack.